libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/api/ElasticsearchEntitlementChecker.java

```diff
@@ -61,10 +91,6 @@ private static boolean isTriviallyAllowed(Module requestingModule) {
             logger.debug("Trivially allowed: Entire call stack is in the boot module layer");
```

Comment:
This log message doesn't seem right anymore. It's more like the entire call stack is composed of system modules?